LoopKitUI/Views/Settings Editors/DeliveryLimitsEditor.swift

```diff
@@ -379,13 +392,39 @@ public struct DeliveryLimitsEditor: View {
                 )
             )
         case .saveError(let error):
-            return Alert(
-                title: Text(LocalizedString("Unable to Save", comment: "Alert title when error occurs while saving a schedule")),
-                message: Text(error.localizedDescription)
-            )
+            switch error {
+            case SaveTherapySettingsError.cancelTempBasalError(let error):
+                return cancelTempBasalAlert(error)
+            default:
+                return SwiftUI.Alert(
+                    title: Text(LocalizedString("Unable to Save", comment: "Alert title when error occurs while saving a schedule")),
+                    message: Text(error.localizedDescription)
+                )
+            }
         }
     }
 
+    private func cancelTempBasalAlert(_ error: Error?) -> SwiftUI.Alert {
+        Alert(
+            title: Text(LocalizedString("Failed to Cancel Temp Basal", comment: "Alert title for failing to cancel temp basal")),
+            message: Text(String(format: LocalizedString("%@Tidepool Loop was unable to cancel your current temporary basal rate, which is higher than the new Max Basal limit you have set. This may result in higher insulin delivery than desired.\n\nConsider suspending insulin delivery manually and then immediately resuming to enact basal delivery with the new limit in place.",
```

Comment:
App specific string here. Would recommend, as per the above comment, that this special handling for the error be removed and instead, have the hosting app return a sufficiently descriptive error type.

Reply:
As above, see the bug.  There's a different alert title and content.

Reply:
Moved content to `DeviceDataManager` in `Loop`